softwarereview_author.Rmd

```diff
@@ -10,7 +10,11 @@ This concise guide presents the software peer review process for you as a packag
     -   If you would like your package to also be submitted to
         [Journal of Open-Source Software](http://joss.theoj.org/) (JOSS), it should include a `paper.md` file describing the package. More detail on JOSS's requirements can be found [at their website](http://joss.theoj.org/about#author_guidelines).
     -   If you choose this option you should *not* submit your package to JOSS separately. It will be evaluated by JOSS based on the rOpenSci review.
--   We strongly suggest submitting your package for review _before_ publishing on CRAN or submitting a software paper describing the package to a journal. Review feedback may result in major improvements and updates to your package, including renaming and breaking changes to functions.
+- Please do not submit your package too early or too late in its development.
```

Comment:
```suggestion
- Please consider the best time in your package's development to submit.  Your package should be sufficiently mature so that reviewers are able to review all essential aspects, but keep in mind that review may result in major changes.
```